### ISSUE TITLE
Make website accessible

### ISSUE THREAD

**Issue by stefandesu** [gbv/coli-conc-next#5]
The website should meet accessibility standards. Here are a few resources:

- [10 Ways to Make Your Website Accessible](https://www.dreamhost.com/blog/make-your-website-accessible/)
- https://developers.google.com/web/fundamentals/accessibility
- Color contrast checkers:
  - https://webaim.org/resources/contrastchecker/
  - https://contrastchecker.com

**Comment by stefandesu** [comment]
This is also a useful tool: https://wave.webaim.org/report#/https://gbv.github.io/coli-conc-next/ (note that the tool messes up the alternating backgrounds and this causes some misinterpretation of contrast issues)

I eliminated all obvious accessibility issues (missing alt texts and color contrast issues in particular), but it's not perfect. The biggest remaining issue is making the menu on mobile accessible. Currently, it is not properly accessible using the keyboard only, although using keyboard navigation together with a screen reader should actually work fine (the keyboard tabs through the hidden menu items, but it's currently not possible to show/hide the menu using  the keyboard). Also, the software table under Publications has some smaller issues still.
